packages/core/src/runner/runner.ts

```diff
@@ -96,30 +95,22 @@ function execute(input: unknown, callbacks: Record<string, Callback[]>, jsonPath
     return;
   }
 
-  try {
-    const nimma = new Nimma(jsonPathExpressions, {
-      fallback: jsonPathPlus,
-      unsafe: false,
-      output: 'auto',
-    });
-
-    nimma.query(
-      input,
-      Object.entries(callbacks).reduce<Record<string, Callback>>((mapped, [key, cbs]) => {
-        mapped[key] = scope => {
-          for (const cb of cbs) {
-            cb(scope);
-          }
-        };
-
-        return mapped;
-      }, {}),
-    );
-  } catch (e) {
-    if (isAggregateError(e) && e.errors.length === 1) {
-      throw e.errors[0];
-    } else {
-      throw e;
-    }
-  }
```

Comment:
in preparation for https://github.com/stoplightio/spectral/tree/feat/cli%2Frich-errors